src/ast.ml

```diff
@@ -1,10 +1,20 @@
+type attributes =
+  {
+    id: string option;
+    classes: string list;
+    attributes: (string * string) list;
+  }
+
 type 'a link_def =
   {
     label: 'a;
     destination: string;
     title: string option;
+    attributes: attributes;
   }
```

Comment:
Can we put this in its own module `Link_def.t` ?

Reply:
I just thought the same when moving attributes to his own module, felt a bit off to have it alone, done now

Reply:
Just one question, when the type checker does not know the fields for the record creation:
eg:
```
File "src/parser.ml", line 1507, characters 7-16:
1507 |       {label; destination; title = None; attributes = Attributes.empty}
              ^^^^^
Error: Unbound record field label
```
Do you prefer that I force the type of the variable on the function to `Ast.Link_def.t`, or that I use `Ast.Link_def.label instead` of `label` ?

Reply:
The usual way is to explicitly qualify the label with the module name, i.e. `Link_def.label`.

Reply:
Okay, I'll do that then